en/controllers/request-response.rst

```diff
@@ -48,7 +48,7 @@ The request exposes routing parameters through the ``getParam()`` method::
 
 To get all routing parameters as an array use ``getAttribute()``::
 
-    $params = $this->request->getAttribute('params');
+    $parameters = $this->request->getAttribute('params');
```

Comment:
I find the status quo for variables here acceptable, especially with it reading a "params" key.
I am OK for all the textual representations, however.